libs/MeshKernel/include/MeshKernel/BilinearInterpolationOnGriddedSamples.hpp

```diff
@@ -48,7 +49,7 @@ namespace meshkernel
                                               UInt numYCoord,
                                               const Point& origin,
                                               double cellSize,
-                                              const std::vector<double>& values);
+                                              const void* values);
```

Comment:
Is is possible ti use `const std::vector<ValueType>& values` instead?

Reply:
Should cellSize and perhaps even the members of Point class also be of ValueType?

Reply:
Re. Ahmads comment, I think the values should be at least: `const ValueType* values`.

Reply:
This is a guess, @lucacarniato  will need to confirm.
One reason for this Jira item is to reduce the memory footprint, so larger data sets can be handled. If a vector is to be passed then a copy of the original data has to be made, increasing memory usage, since it will be passed to the api function as a raw pointer.

Reply:
Used std::span